server/models/SiteMetadataModel/index.js

```diff
@@ -0,0 +1,17 @@
+import { collections } from '../../utils/mongoCollections'
+
+const SiteMetadataModel = {
+  update: async (db, query, metadataAttributes, metadataParticipants) =>
```

Comment:
should we call this method `upsert` since that's what we're doing?

Reply:
[c1cf0d0](https://github.com/AMP-SCZ/dpdash/pull/382/commits/c1cf0d04b9d9dd43b6aceef72cc4f1f45af67a9c)